### DISCUSSION TITLE
Sensortypes voor aanwezigheid of nabijheidssensoren?

### DISCUSSION THREAD

**Discussion by tmeindertsma** [kadaster-labs/sensrnet-home#268] [Q&A] [answered]
Ik ben gevraagd om voor een project van de gemeente Amsterdam (waarbij verkeersbewegingen op het water gemonitord worden met behulp van verschillende types sensoren) de sensoren toe te voegen aan het sensoren register. Ik heb hiervoor van de privacy officer een bulk-excel ontvangen. Voor een aantal sensoren weet ik niet 100% zeker onder welk type ze vallen. Vai via werd ik voor deze vraag doorverwezen naar deze github.

Het gaat om deze sensoren:
1. Passieve infrarood sensoren: detecteren passerende objecten
2. RFID sensoren (radio-frequency identification): detecteren de chip-nummers die geïntegreerd zijn in vignetten op passerende vaartuigen.

Ik ga ervan uit dat beide vallen onder de categorie "aanwezigheid of nabijheidsensoren". Is dat correct? Vooral over de RFID-sensoren twijfel ik, omdat ze naast aanwezigheid van een vaartuig ook een uniek kenmerk (het chip-nummer) detecteren.

**Comment by ewalddijkstra** [comment]
De vraag is wat de belangrijkste functie van de sensor is. Als dat het detecteren van objecten is, dan zijn het "aanwezigheid of nabijheidsensoren". Dat neemt niet weg dat de sensor meerdere functies kan hebben. Zo heeft een ANPR camera voor de millieuzone primair een functie in de handhaving, maar tevens een functie binnen verkeersonderzoek. De RFID sensor zou daarom ook kunnen worden beschouwd als toegangsdetectie.

**Comment by robhaans** [comment, marked as answer]
Het gaat om de (belangrijkste) functie. De gebruikte techniek zegt wel wat, maar niet alles. Zowel infrarood als rfid kunnen voor verschillende functies gebruikt worden. In jouw voorbeeld lijken de infrarood sensoren daadwerkelijk voor nabijheidsdetectie gebruikt te worden. De RFID worden in jouw voorbeeld alleen gebruikt voor identificatie. Mogelijk wordt er na de identificatie ook nog dingen gedaan (authenticatie, betaling, etc.), maar identificatie lijkt van de sensor de belangrijkste functie.

**Reply by tmeindertsma** [reply]
De belangrijkste functie van zowel de infrarood en de RFID sensoren is het monitoren van drukte, dus het detecteren van passages. Secundair worden de RFID sensoren potentieel nog gebruikt voor identificatie (zoals bepalen of binnenhavengeld betaald is en of een vaartuig betrokken is bij geluidsoverlast), maar deze functionaliteit ligt nog niet vast.
Op basis van dit antwoord en het antwoord van ewalddijkstra is de beste categorisering van beide types sensoren "aanwezigheid of nabijheidsensoren".
